code/datums/holocall.dm

```diff
@@ -636,51 +636,83 @@
 	SAY All of these attributes contribute to an overall Agent Level, which is mostly involved in fear-related interactions.
 	DELAY 50"}
 
-/obj/item/disk/holodisk/tutorial/lobcorp2
+/obj/item/disk/holodisk/tutorial/info1
 	preset_record_text = {"
 	NAME Angela
 	DELAY 10
 	SAY Lobotomy Corporation is an energy corporation, and we produce energy by working on entities called "Abnormalities".
 	DELAY 45
-	SAY Working also improves your attributes, allowing you to become a better employee.
-	DELAY 35
 	SAY There are four primary types of work: Instinct, Insight, Attachment, and Repression.
 	DELAY 40
-	SAY Instinct work increases your Fortitude. It represents nourishment and physical contact.
-	DELAY 40
-	SAY Insight work increases your Prudence. It represents improving the environment of the Containment Unit.
-	DELAY 40
-	SAY Attachment work increases your Temperance. It represents satisfying social needs through communication.
-	DELAY 40
-	SAY Repression work increases your Justice. It represents suppressing and controlling impulses and desires.
-	DELAY 40
-	SAY The success chance of each work type is listed as Very Low/Low/Common/High/Very High.
+	SAY The <span class='bold'>success chance</span> of each work type is listed as Very Low/Low/Common/High/Very High. This can be adjusted work modifiers such as your Temperance.
```

Comment:
```suggestion
	SAY The <span class='bold'>success chance</span> of each work type is listed as Very Low/Low/Common/High/Very High. This can be adjusted by work modifiers such as your Temperance.
```